src/manager.js

```diff
@@ -103,14 +103,15 @@ AdManager.prototype.initGoogleTag = function() {
 */
 AdManager.prototype.initAmazonA9 = function() {
   try {
-    this.amznads = amznads;
+    amznads.apiReady;
   } catch(e) {
     if(e.name == "ReferenceError") {
       this.amznads = false;
       console.log('bulbs-public-ads-manager: amznads is not defined');
     }
   }
   if(this.amznads) {
```

Comment:
Is the intention to just catch errors here, or do we need to wait until amazon ads to be ready with some sort of async function?